README.md

```diff
@@ -1,8 +1,8 @@
-# kogibot
+# kiebot
 
-![Kogibot - Java 11](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-11.yml/badge.svg) 
-![Kogibot - Java 16](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-16.yml/badge.svg)
-![Kogibot - Java 17](https://github.com/spolti/kogibot/actions/workflows/main-merge-java-17.yml/badge.svg)
+![Kiebot - Java 11](https://github.com/spolti/kiebot/actions/workflows/main-merge-java-11.yml/badge.svg) 
```

Comment:
for the badges we can use the url from the current jog, pull_request I guess

Reply:
yep

Reply:
updated